README.md

```diff
@@ -238,23 +238,79 @@ Registry](https://github.com/stackbuilders/hapistrano/pkgs/container/hapistrano)
 
 Check the documentation [here](.github/workflows/README.md)
 
-## Nix
+## Development
 
-If you want to use Nix for building Hapistrano, the required release.nix and default.nix are available.
+### Requirements
 
-For installing the hap binary in your local path:
-```bash
-nix-env -i hapistrano -f release.nix
+- Install [Zsh](https://www.zsh.org/)
+- Use [GHCup][ghcup] to install:
+  - GHC 8.10.x or 9.0.x (it is recommended to try both for backward
+    compatibility)
+  - Cabal 3.x
+
+Alternatively, install only [Nix](https://nixos.org/download.html) and enable
+[Flakes](https://nixos.wiki/wiki/Flakes).
```

Comment:
@sestrella I think we should enable Flakes in a permanent way, right? Does it make sense to specify that here?